boefjes/boefjes/plugins/kat_nuclei/main.py

```diff
@@ -1,22 +1,27 @@
 from typing import Tuple, Union, List
-
 import docker
-
 from boefjes.job_models import BoefjeMeta
 
+
 NUCLEI_IMAGE = "projectdiscovery/nuclei:latest"
 
 
+def verify_hostname_meta(input):
+    # if the input object is HostnameHTTPURL then the hostname is located in netloc
+    if "netloc" in input and "name" in input["netloc"]:
+        netloc_name = input["netloc"]["name"]
+        port = input.get("port")
```

Comment:
This could also return a `None`, which makes the formatted string look like `<hostname>:None`. It can be avoided by giving a default port, e.g. `port = input.get("port", 80)`

But since the `port` property is required and therefore guaranteed to exist, you can simply access that element directly: `port = input["port"]`
```suggestion
        port = input["port"]
```